lib/transit_data/glides_report/util.ex

```diff
@@ -20,16 +20,17 @@ defmodule TransitData.GlidesReport.Util do
     |> String.pad_leading(count, "0")
   end
 
-  @spec group_by_hour(Enumerable.t({stop_id :: String.t(), timestamp :: integer})) :: %{
```

Comment:
Did you mean to remove this function? I'm not sure if we were keeping the ability to group by hour.

I still see it this referenced in /reports/glides_terminal_departure_accuracy.livemd though

Reply:
Oof thank you, I was focused on the notebook that I was editing and not the other one.

I did mean to remove the function—the logic for this became much simpler after I started using the `Departure` struct, so [just calling `Enum.group_by` directly sufficed](https://github.com/mbta/transit_data_reports/blob/0484b78cee3e6c5e46b1a7c527139c835e866791/reports/glides_terminal_missed_extraneous_predictions.livemd?plain=1#L382-L383).

I'll catch the other notebook up with the changes I made in the first one.

(Would love to dedup some more code between them, but most of what remains is presentational and would require adding Kino to the mix project's deps, which seems like not the right way to go.)